### PR TITLE
OSDEV-1691:add FlapDetection parameter to status board status updates model

### PR DESCRIPTION
**Pull request by nesty156**
Hey, I need to add flap detection parameter to the status board updates model for my future MR on this ticket: [OSDEV-1691](https://issues.redhat.com/browse/OSDEV-1691)

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.396 Sep 23 2024
+- Add `FlapDetection` field to `status_board` `status_updates` model
```
Comment (tzvatot) [inline CHANGES.md:6]: Better rephrase as "status_board.status_updates" as it's more descriptive.

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.396 Sep 23 2024
+- Add `FlapDetection` field to `status_board.status_updates` model
+
 ## 0.0.395 Sep 13 2024
 - Add `WifEnabled` to `Version` model
+- Add `AWSShard` to the `ProvisionShard` model
```
Comment (tzvatot) [inline CHANGES.md:10]: This does not seems to be related to the flap detection. Is your code rebased?

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 2 above)
Comment (nesty156) [reply CHANGES.md:10]: sorry.

File: CHANGES.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.396 Sep 23 2024
```
Comment (tzvatot) [inline CHANGES.md:5]: Do you mind fixing the date? since we are going to create the release today...

**Review by tzvatot — CHANGES_REQUESTED** [review]

**Comment by tzvatot** [comment]
LGTM

**Pull request merged**